lib/iris/_data_manager.py

```diff
@@ -264,6 +277,24 @@ def _deepcopy(self, memo, data=None, fill_value='none',
 
         return result
 
+    def _propagate_masked_data_fill_value(self):
+        """
+        Align the data manager fill-value with the real masked array
+        fill-value.
+
+        """
+        data = self._real_array
+        if ma.isMaskedArray(data):
```

Comment:
This logic check is duplicated from [L77](https://github.com/bjlittle/iris/blob/638c0c665a3945cfeb72e8550a82c148f9a39138/lib/iris/_data_manager.py#L77) and [L422](https://github.com/bjlittle/iris/blob/638c0c665a3945cfeb72e8550a82c148f9a39138/lib/iris/_data_manager.py#L422). As this is a private method we would not expect it to be called outside of this class, so I believe that we do not need this line (we can assume `data` always will be masked).